### PR TITLE
Feature/ Component

### PR DESCRIPTION
**Pull request by V0pe**
### Component: Props & State

In this work, I did the following to render the component. 
- [ ] remove boiler plate
- [ ] set up the component
- [ ] Add the markup and styles

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import Calculator from './Calculator';
 
 test('renders learn react link', () => {
-  render(<App />);
```
Comment (Mihndim2020) [inline src/App.test.js]: - [ ] Please your project is missing an `App.js` component, please kindly add that, and make sure is is class-based as [requested](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, `Use class based components (All components should be class-based).` " `Once the Calculator is ready, import it and use it in your main component (App.js).`, ". After adding the `App.js` component, then you import the `Calculator.js` component into the `App.js`  component and then the `App.js`  component into the `index.js` file.

File: src/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import './Calculator.css';
+
+function Calculator() {
```
Comment (Mihndim2020) [inline src/Calculator.js]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, " `Use class based components (All components should be class-based).`", kindly change your `Calculator component`, which is currently a `functional component` to a `class-based` component.

File: src/index.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 import './index.css';
-import App from './App';
+import Calculator from './Calculator';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (Mihndim2020) [inline src/index.js]: - [ ] Please here, you will need to render the `App` component, not the `Calculator component` as explained above.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @V0pe, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Great job with Gitflow. 

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below and under the review._

 #### I will leave the comment here, since your `Readme file` is not accessible for this PR ####

- [ ] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your poject.  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.test.js (hunk 1 above)
Comment (V0pe) [reply src/App.test.js]: @Mihndim2020 Thanks for the insight. I have used class based components now. and imported the calculator.js into the app

File: src/index.js (hunk 3 above)
Comment (V0pe) [reply src/index.js]: @Mihndim2020 I have made the changes

**Review by mikethreels — APPROVED** [review]
Hi Victor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**